dashboard/app/views/devise/shared/_oauth_links.haml

```diff
@@ -22,6 +22,6 @@
         consult the "Developer OAuth Setup on Localhost" doc in the Engineering drive folder
 
     - [:google_oauth2, :microsoft_v2_auth, :facebook, :clever].each do |provider|
-      = button_to omniauth_authorize_path(resource_name, provider), id: "#{provider}-sign-in", class: "oauth-sign-in with-#{provider}" do
+      = button_to omniauth_authorize_path(resource_name, provider, script_name: nil), id: "#{provider}-sign-in", class: "oauth-sign-in with-#{provider}" do
```

Comment:
Removes the /global/fa prefix from the OAuth path, which is added in the Global Edition middleware here:
https://github.com/code-dot-org/code-dot-org/blob/7d60477caf63b1a0b2149bce547f995a94762223/lib/cdo/rack/global_edition.rb#L64